src/styles/calendar.scss

```diff
@@ -0,0 +1,44 @@
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  gap: $cell-space;
+  width: $cal-width;
+  padding: $cal-padding;
+
+  &__day {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    cursor: pointer;
+    border: 1px solid black;
+    height: $cell-size;
+    width: $cell-size;
+    background-color: $bg-color-day;
+    transition-duration: 0.5s;
```

Comment:
Let's specify the property, for which you'll use transition duration
If you don't specify the property, it will affect in a bad way on the website performance

Reply:
done